quickstart.md

```diff
@@ -30,18 +32,18 @@ To add the Pravega connector dependencies to you project, add the following entr
 </dependency>
 ```
 
-Use appropriate version as necessary. The snapshot versions are published to [jcenter](https://oss.jfrog.org/artifactory/jfrog-dependencies/io/pravega/pravega-connectors-flink_2.11/) repository and the release artifacts are available in [Maven Central](https://mvnrepository.com/artifact/io.pravega/pravega-connectors-flink_2.11) repository.
+Use appropriate version as necessary. The snapshot versions are published to [`jcenter`](https://oss.jfrog.org/artifactory/jfrog-dependencies/io/pravega/pravega-connectors-flink_2.11/) repository and the release artifacts are available in [`Maven Central`](https://mvnrepository.com/artifact/io.pravega/pravega-connectors-flink_2.11) repository.
 
-Alternatively, you could build and publish the connector project to local maven repository by following below steps and make use of that version as your application dependency.
+Alternatively, we could build and publish the connector project to local maven repository by following the below steps and make use of that version as your application dependency.
 
 ```
 ./gradlew clean install
 ```
 
-## Running / deploying the application
+## Running / Deploying the Application
 
-From Flink's perspective, the connector to Pravega is part of the streaming application (not part of Flink's core runtime), so the connector code must be part of the application's code artifact (JAR file).   Typically, a Flink application is bundled as a _fat-jar_ (also known as an _uber-jar_) such that all its dependencies are embedded.
+From Flink's perspective, the connector to Pravega is part of the streaming application (not part of Flink's core runtime), so the connector code must be part of the application's code artifact (JAR file). Typically, a Flink application is bundled as a _`fat-jar`_ or _`uber-jar_`, such that all its dependencies are embedded.

```

Comment:
Let's use the original format  

_fat-jar_ (also known as an _uber-jar_)

Reply:
Accepted suggestion